lib/cn/kblock_builder.c

```diff
@@ -137,6 +137,7 @@ struct curr_kblock {
 
     uint64_t total_key_bytes;
     uint64_t total_val_bytes;
+    uint64_t total_vused_bytes;
```

Comment:
are you going to add this to the REST output?

Reply:
No plans for that as of now.